src/content/blog/why-not-effect.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: "Why Val Town is not using Effect"
```

Comment:
Another idea:

```suggestion
title: "Why we're not using Effect"
```

I don't feel strongly either way, just wanted to throw it out there